.gitignore

```diff
@@ -1,13 +1,17 @@
 *.pyc
 *~
-pydrive/test/configs/*
-pydrive/test/credentials/*
+pydrive2/test/configs/*
```

Comment:
better to create `pydrive2/test/.gitignore`

Reply:
makes sense

Reply:
👍